### PR TITLE
feat(drive): improve ABCI logging

### PR DESCRIPTION
**Pull request by lklimek**
## Issue being fixed or feature implemented

Logs in Drive don\t include necessary information about ABCI request.

## What was done?

Updated rs-tenderdash-abci to the most recent version which adds tracing spans to requests.

Cleaned up code a bit to improve logs.

## How Has This Been Tested?

Using local devnet.

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/rs-drive-abci/src/abci/handlers.rs [hunk 1]
```diff
@@ -101,7 +101,7 @@ where
                 .unwrap_or_default(),
         };
 
-        tracing::info!(method = "info", ?request, ?response, "info executed");
+        tracing::info!("info executed");
```
Comment (shumkov) [inline packages/rs-drive-abci/src/abci/handlers.rs:104]: I guess we can remove this one

File: packages/rs-drive-abci/src/abci/handlers.rs [hunk 2]
```diff
@@ -136,7 +133,7 @@ where
 
         let app_hash = hex::encode(&response.app_hash);
 
-        tracing::info!(method = "init_chain", app_hash, "init chain executed");
+        tracing::info!(app_hash, "init chain executed");
```
Comment (shumkov) [inline packages/rs-drive-abci/src/abci/handlers.rs:136]: The same

File: packages/rs-drive-abci/src/abci/handlers.rs [hunk 3]
```diff
@@ -720,7 +715,7 @@ where
             height: self.platform.state.read().unwrap().height() as i64,
             codespace: "".to_string(),
         };
-        tracing::trace!(method = "query", ?request, ?response);
+        tracing::trace!("query executed");
```
Comment (shumkov) [inline packages/rs-drive-abci/src/abci/handlers.rs:718]: Remove this one

File: packages/rs-drive-abci/src/execution/check_tx/v0/mod.rs [hunk 4]
```diff
@@ -251,7 +248,11 @@ mod tests {
             )
             .expect("expected to process state transition");
 
-        dbg!(result);
+        tracing::trace!(
```
Comment (shumkov) [inline packages/rs-drive-abci/src/execution/check_tx/v0/mod.rs:251]: I guess we should remove this one

File: packages/rs-drive-abci/src/logging.rs [hunk 5]
```diff
@@ -669,6 +669,26 @@ impl Logger {
...
+/// Helper that initializes logging in unit tests
+///
+///
+/// For verbosity, see drive-abci --help or use 0 or 5
+pub fn init_for_tests(verbosity: u8) {
+    let mut logger_builder = LogBuilder::new();
+    let config = LogConfig {
+        destination: "stderr".to_string(),
+        verbosity,
+        color: None,
```
Comment (shumkov) [inline packages/rs-drive-abci/src/logging.rs:681]: Colors would be nice ))

**Review by shumkov — CHANGES_REQUESTED** [review]

File: packages/rs-drive-abci/src/logging.rs (hunk 5 above)
Comment (lklimek) [reply packages/rs-drive-abci/src/logging.rs:681]: None means it will be autodetected

File: packages/rs-drive-abci/src/abci/handlers.rs (hunk 2 above)
Comment (lklimek) [reply packages/rs-drive-abci/src/abci/handlers.rs:136]: This adds app hash, and it is in "info" level. rs-tenderdash-abci logs request/response on "trace" level. I believe it's worth keeping it here for debugging.

**Review by shumkov — APPROVED** [review]
Great! 👍

File: packages/rs-drive-abci/src/abci/handlers.rs (hunk 2 above)
Comment (shumkov) [reply packages/rs-drive-abci/src/abci/handlers.rs]: Agree!

**Review by markin-io — APPROVED** [review]

**Pull request merged**